mkdocs.yml

```
@@ -163,14 +155,22 @@ nav:
         - Audit Log: ui/audit-log.md
         - Troubleshooting: ui/troubleshooting.md
         - Prefect Cloud API Rate Limits: ui/rate-limits.md
+    - Guides: 
+        - guides/index.md
+        - Testing: guides/testing.md
+        - Dask & Ray: guides/dask-ray-task-runners.md
+        - Deployment:
+            - Docker: guides/deployment/docker.md
+            - Azure Container Instance: guides/deployment/cloud-agent.md
```

I messed up the link:
```suggestion
            - Azure Container Instance: guides/deployment/aci.md
```